01-messaging.md

```diff
@@ -258,10 +258,19 @@ The `features` field MUST be padded to bytes with 0s.
     1. type: 1 (`networks`)
     2. data:
         * [`...*chain_hash`:`chains`]
-
+    1. type: 3 (`compression_algorithms`)
+    2. data:
+        * [`...*byte`:`supported_algorithms`]
 
 The optional `networks` indicates the chains the node is interested in.
 
+The optional `compression_algorithms` is a bitfield indicating what compression
+algorithms the node supports for [gossip queries](07-routing-gossip.md#query-messages).
+
+| Bit Position  | Algorithm                                            |
+| ------------- | ---------------------------------------------------- |
+| 0             | [zlib_deflate](https://www.ietf.org/rfc/rfc1950.txt) |
```

Comment:
Carry over from spec meeting:
  * start out w/ 2 bits 
  * bit `0` is uncompressed 
  * bit `1` is zlib

Reply:
Done in https://github.com/lightningnetwork/lightning-rfc/pull/825/commits/5071b933d97fb7eb53f5505874c4f02d78e14028